dms/models/access_groups.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017-2019 MuK IT GmbH
+# Copyright 2020 RGB Consulting
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class DmsAccessGroups(models.Model):
+    _name = "dms.access.group"
+    _description = "Record Access Groups"
+
+    _parent_store = True
+    _parent_name = "parent_group_id"
+
+    name = fields.Char(string="Group Name", required=True, translate=True)
+    parent_path = fields.Char(string="Parent Path", index=True)
+    perm_read = fields.Boolean(string="Read Access")
+    perm_create = fields.Boolean(string="Create Access")
+    perm_write = fields.Boolean(string="Write Access")
+    perm_unlink = fields.Boolean(string="Unlink Access")
+    directory_ids = fields.Many2many(
+        comodel_name="dms.directory",
+        relation="dms_directory_groups_rel",
+        string="Directories",
+        column1="gid",
+        column2="aid",
+        readonly=True,
+    )
+    count_users = fields.Integer(compute="_compute_users", string="Users", store=True)
+    count_directories = fields.Integer(
+        compute="_compute_count_directories", string="Count Directories"
+    )
+
+    @api.depends("directory_ids")
+    def _compute_count_directories(self):
+        for record in self:
+            record.count_directories = len(record.directory_ids)
+
+    @api.model
+    def _add_magic_fields(self):
```

Comment:
Since no model inherits from this, is this really needed?

Reply:
Yes, the original module was intended to be inherited, but not here